tests/Endpoint_selenium_unittest.py

```diff
@@ -0,0 +1,155 @@
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+import unittest
+import re
+import sys
+import os
+
+
+class EndpointTest(unittest.TestCase):
+    def setUp(self):
+        # Initialize the driver
+        # When used with Travis, chromdriver is stored in the same
+        # directory as the unit tests
+        self.driver = webdriver.Chrome('chromedriver')
+        # Allow a little time for the driver to initialize
+        self.driver.implicitly_wait(30)
+        # Set the base address of the dojo
+        self.base_url = "http://localhost:8000/"
+        self.verificationErrors = []
+        self.accept_next_alert = True
+
+    def login_page(self):
+        # Make a member reference to the driver
+        driver = self.driver
+        # Navigate to the login page
+        driver.get(self.base_url + "login")
+        # Good practice to clear the entry before typing
+        driver.find_element_by_id("id_username").clear()
+        # These credentials will be used by Travis when testing new PRs
+        # They will not work when testing on your own build
+        # Be sure to change them before submitting a PR
+        driver.find_element_by_id("id_username").send_keys(os.environ['DD_ADMIN_USER'])
+        driver.find_element_by_id("id_password").clear()
+        driver.find_element_by_id("id_password").send_keys(os.environ['DD_ADMIN_PASSWORD'])
+        # "Click" the but the login button
+        driver.find_element_by_css_selector("button.btn.btn-success").click()
+        return driver
+
+    # first thing first. We create product, just to make sure there is atleast 1 product available
+    # to assign endpoints to when creating or editing any.
+    def test_create_product(self):
```

Comment:
Try to find a way to import this test from the Product unit test instead of copy/paste.

Reply:
this is done. I had to find a way to import the test file as module.